src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getLinkDataCy(index: number, totalPages: number): string {
+  if (index === 0) {
+    return 'prevLink';
+  } else {
+    return index === totalPages + 1 ? 'nextLink' : 'pageLink';
+  }
+}
+
+export function getLinkHref(index: number, totalPages: number): string {
+  if (index === 0) {
+    return '#prev';
+  } else {
+    return index === totalPages + 1 ? '#next' : `#${index}`;
```

Comment:
As with the `getLinkDataCy` function, using zero-based indexing for `totalPages` would make the code more consistent with common practices. If there's a specific reason for using one-based indexing, ensure it's well-documented.